### DISCUSSION TITLE
Do we have inbuilt classes for the Appium-flutter-driver as like as the Inbuilt classes in the Native web driver.

### DISCUSSION THREAD

**Discussion by VenkateshSeenivasan** [appium/appium#18709] [Q&A]
In Appium driver we have some inbuilt classes available which helps to automate the Native application(iOS or android). By using the Java-client as a dependency in the Maven project.

In order to automate Flutter Application we need to use the Appium-flutter-driver.

But, In Appium-flutter-driver we were not able to find the inbuilt classes and can’t able to integrate the Appium-flutter-driver in maven project. Because we don’t have Maven dependency for integrating the Appium-flutter-driver into the maven project.
Could you please share the related documentation to us If available.

**Comment by VenkateshSeenivasan** [comment]
@jlipps could you please have a look ⬆️ . It would be helpful.

**Reply by jlipps** [reply]
I'm sorry, I don't really understand this question. Maybe a java client maintainer like @SrinivasanTarget would know better?

**Comment by VenkateshSeenivasan** [comment]
Hi @SrinivasanTarget could you please take a look. It would be helpful.

**Reply by vnuta9** [reply]
@saikrishna321 @VenkateshSeenivasan 
CC: @VenkateshSeenivasan 
I have the same request, will you please respond on this topic ?
